deploy/charts/harvester/values.yaml

```
@@ -27,7 +27,7 @@ kubevirt-operator:
     operator:
       image:
         repository: registry.suse.com/harvester-beta/virt-operator
```

```suggestion
        repository: registry.suse.com/suse/sles/15.3/virt-operator
```
hey, just dont forget to update the registry path: `harvester-beta` -> `suse/sles/15.3`